signer/src/transaction_coordinator.rs

```diff
@@ -827,6 +859,66 @@ where
         }
     }
 
+    fn authenticate_message(
+        packet: &wsts::net::Packet,
+        public_keys: &hashbrown::HashMap<u32, p256k1::point::Point>,
+        public_key_point: p256k1::point::Point,
+        sender_is_coordinator: bool,
+    ) -> bool {
+        let check_signer_public_key = |signer_id| match &public_keys.get(&signer_id) {
```

Comment:
You don't need the reference before `public_keys` here.

Reply:
Done